be/src/exec/pipeline/exchange/exchange_sink_operator.h

```diff
@@ -177,6 +180,8 @@ class ExchangeSinkOperatorFactory final : public OperatorFactory {
     bool _enable_exchange_pass_through;
 
     FragmentContext* const _fragment_ctx;
+
+    const std::vector<int32_t>& _output_columns;
```

Comment:
_output_columns should be value in ExchangeSinkOperatorFactory, and reference in ExchangeSinkOperator